client/src/app/site/assignments/assignment-list/assignment-list.component.ts

```diff
@@ -28,11 +28,10 @@ export class AssignmentListComponent extends ListViewBaseComponent<ViewAssignmen
         titleService: Title,
         translate: TranslateService,
         matSnackBar: MatSnackBar,
-        private repo: AssignmentRepositoryService
+        public repo: AssignmentRepositoryService
```

Comment:
should stay private. You should not access the repo from inside a html.
Make a function to forward.